apps/meteor/app/livechat/server/methods/saveInfo.js

```diff
@@ -7,6 +7,7 @@ import { LivechatRooms } from '../../../models/server';
 import { callbacks } from '../../../../lib/callbacks';
 import { Livechat } from '../lib/Livechat';
 
+// TODO: Deprecate this method in favor of "livechat/room.saveInfo" endpoint in v6.0.0
```

Comment:
You can use the JSDoc deprecation notation for this 
https://jsdoc.app/tags-deprecated.html

Reply:
That was helpful, thanks!